biz.aQute.bndlib/src/aQute/bnd/osgi/Instruction.java

```diff
@@ -64,6 +64,10 @@ public boolean accept(File pathname) {
 	private boolean					optional;
 
 	public Instruction(String input) {
+
+		if (input == null || input.isEmpty())
```

Comment:
I am not sure this is the right fix for #2544 (You referenced a different issue number in the commit comment). I assume when they say `classifier=` they want to match classifier having no value not any value which is what this change would mean. For empty string, I would think the pattern would be `!*`: Match nothing. Passing null should be a programming error, not match all.